src/Spago/Bower.hs

```diff
@@ -65,23 +65,18 @@ writeBowerJson limitJobs dryRun = do
 
   ignored <- Git.isIgnored bowerPath
   when ignored $ do
-    die $ bowerPath <> " is being ignored by git - change this before continuing."
+    die $ bowerPath <> " is being ignored by git - change this before continuing"
 
-  case dryRun of
-    DryRun -> echo $ "Skipped writing " <> bowerPath <> " because this is a dry run."
-    NoDryRun -> do
-      liftIO $ ByteString.writeFile bowerPath bowerJson
-      echo $ "Generated " <> bowerPath <> " using the package set."
+  liftIO $ ByteString.writeFile bowerPath bowerJson
```

Comment:
No strong feelings, but maybe writing `bower.json` should also be a `DryAction` ?

Maybe not because it is easy to undo, but then so is creating a Git tag (unless it has been pushed elsewhere).

Reply:
Yeah I agree. My rationaly for doing this was: what if one wants to take a look at the generated file before creating a tag?

On the other hand also creating a tag is easy to undo, so I'll make everything dry and clarify the log message for the tag creation